pkg/operator/workload/workload_openshiftapiserver_v311_00_test_sync.go.BAK

```diff
@@ -100,17 +100,18 @@ func TestAPIServerDeploymentProgressingCondition(t *testing.T) {
 			openshiftConfigClient := configfake.NewSimpleClientset()
```

Comment:
`.BAK`?  Dejavú of MS-DOS times ;-)

Reply:
I still need to check if the tests make sense or rewrite them :)